pkg/crc/machine/machine.go

```diff
@@ -267,6 +267,6 @@ func setMachineLogging(logs bool) error {
 }
 
 func unsetMachineLogging() {
-	logging.CloseLogFile()
+	logging.CloseLogfile()
```

Comment:
why not `CloseLogFile`?

Reply:
because there's a `OpenLogfile()`

Reply:
`LogFile` makes more sense... a File for Logging... a LogFile... right? It consist of two words: https://en.wikipedia.org/wiki/Log_file and not a single word. The convention is to use Camel Casing...